frontend/src/components/task/task-status-icons.js

```diff
@@ -0,0 +1,99 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import {
+  Redeem as RedeemIcon,
+  Lock as PrivateIcon,
+  LockOpen as PublicIcon
+} from '@material-ui/icons'
+import {
+  Paper,
+  Typography
+} from '@material-ui/core'
+
+const styles = {
+  IconStyle: {
+    fontSize: 80
```

Comment:
@ShubhamCanMakeCommit can we reduce the icon to 48?
```suggestion
    fontSize: 48
```